TestingLowerBounds/ForMathlib/Integrable_of_empty.lean

```diff
@@ -0,0 +1,27 @@
+import Mathlib.MeasureTheory.Integral.Bochner
+
+open MeasureTheory
+
+open Real MeasureTheory Filter MeasurableSpace
+open scoped ENNReal NNReal Topology BigOperators
+
+--TODO: put this in mathlib, maybe just after:
+#check Integrable.of_finite
+
+@[simp]
+lemma Integrable.of_empty {α β : Type*} [MeasurableSpace α] [NormedAddCommGroup β] [IsEmpty α]
+    (f : α → β) (μ : Measure α) : Integrable f μ := Integrable.of_finite μ f
```

Comment:
Is this really useful, since `Integrable.of_finite` exists? I guess this is more discoverable.
We can keep it, but the name should use `isEmpty`, not `empty`:
```suggestion
@[simp]
lemma Integrable.of_isEmpty {α β : Type*} [MeasurableSpace α] [NormedAddCommGroup β]
    [IsEmpty α] (f : α → β) (μ : Measure α) :
    Integrable f μ := Integrable.of_finite μ f
```

Reply:
Also since `Integrable` is in fact `MeasureTheory.Integrable`, you should put those lemmas in the `MeasureTheory` namespace.